### ISSUE TITLE
A user without the EVENT_VIEW user right can see data about events

### ISSUE THREAD

**Issue by roxanamlendea** [hzi-braunschweig/SORMAS-Project#10501]
### Bug Description
After ticket #10346 was fixed , the columns on the Case Directiory related to events are not displayed anymore for users that do not have the EVENT_VIEW user right (View existing events). However, the issue is still reproducible for the columns on the Contact Directory.
Information about Events is still displayed in the following:
- Contact Directory - Detailed view: the 'Latest Event ID' and 'Latest Event Title' columns; + the 'Event: ID, title, description' search field;
- Cases Directory: the 'Event: ID, title, description' search field + the 'Only cases with events' checkbox';
- Dashboard

### Steps to Reproduce

1. Log in with a user without EVENT_VIEW user right (View existing events) user right;
2. Open the Contacts Directory, Cases Directory, Dashboard and  search for the columns and fields related to Events;

### Expected Behavior
- the columns on the Contacts Directory should be hidden for users without the EVENT_VIEW user right;
- the 'Event: ID, title, description' search field and the  'Only cases with events' checkbox' should be read-only for users without the EVENT_VIEW user right; see the example below:
![image](https://user-images.githubusercontent.com/89522688/193278322-f148124e-af4b-47e7-a178-135f9c029cb2.png)
- the Event information on the Dashboard should be hidden for users without the EVENT_VIEW user right;

### Screenshots
![image](https://user-images.githubusercontent.com/89522688/193273955-6a4fd568-d99d-4f48-9db6-4d71e5caef75.png)
![image](https://user-images.githubusercontent.com/89522688/193275225-4093b277-6671-421b-a5d1-de3e74d6b3a4.png)
![image](https://user-images.githubusercontent.com/89522688/193277492-27e3779f-e14e-422f-b45f-5b806db274e6.png)

### System Details

* Device: Windows 10
* SORMAS version: Sormas 1.76.0-SNAPSHOT (3968f00)
* Android version/Browser: Chrome
* Server URL: local environment
* User Role: the user has the user role described in the Additional Information section

### Additional Information
<details>
<summary>Screenshots with the User Rights of the used User Role</summary> 

![image](https://user-images.githubusercontent.com/89522688/193279254-488c7c18-3714-4c7b-a797-5c6cd07fcc76.png)
![image](https://user-images.githubusercontent.com/89522688/193279301-085d10ff-b77e-473f-9d33-6a6f790a7f46.png)
![image](https://user-images.githubusercontent.com/89522688/193279348-0d6637b6-8938-48c0-aa80-a086e76bd80d.png)
![image](https://user-images.githubusercontent.com/89522688/193279398-f504031e-c555-4e41-bc36-975ffa0ee07f.png)
![image](https://user-images.githubusercontent.com/89522688/193279451-d9ccdded-ea47-4c37-9f48-e11e611cea50.png)
![image](https://user-images.githubusercontent.com/89522688/193279501-fa48b26c-5e28-48ce-a432-9cd629f98c8f.png)
![image](https://user-images.githubusercontent.com/89522688/193279547-088061dc-2a70-4f7a-bd2d-c75d91ac6159.png)

**Comment by FredrikSchaeferVitagroup** [comment]
I don't agree with the expected behaviour in the description. I think the forbidden search fields should be hidden instead of disabled. Reasons:
1. There is no way for (non admin) users to enable them.
2. There are so many filter fields. Hiding every bit possible is a good idea to make the filter forms more manageable.

@roxanamlendea Do you have any reasons against mine?

**Comment by roxanamlendea** [comment]
@FredrikSchaeferVitagroup The reason why I wrote in the expected behavior that the 'Event: ID, title, description' search field AND the 'Only cases with events' checkbox should be read-only , is because I saw that there's another checkbox related to Events that's disabled. See the example below:
![image](https://user-images.githubusercontent.com/89522688/194004370-c39aa338-3711-49e5-8fbf-a80fd3b23659.png)
I don't have anything against having them hidden instead of read-only (like you, I agree that it's probably a better idea), but then, for consistency, the checkbox from the screenshot that I posted as an example should be hidden too.

**Comment by FredrikSchaeferVitagroup** [comment]
Then I'll not implement this.

**Comment by roxanamlendea** [comment]
> Then I'll not change this.

Ok, I'll remove it from the description of the ticket

**Comment by abrudanancuta** [comment]
Validated on local env with the version: 1.75.4-SNAPSHOT (89dfbb8)
